internal/agent/event_sender_vortex.go

```diff
@@ -231,7 +231,7 @@ func newEventData(key entity.Key, edata []byte, agentKey string) eventVortexData
 func (s *vortexEventSender) updateLocalMap(entities []identityapi.RegisterEntityResponse) {
 	s.localEntityMap.CleanOld()
 	for _, e := range entities {
-		s.localEntityMap.Put(e.Key, e.ID)
+		s.localEntityMap.Put(entity.Key(e.Name), e.ID)
```

Comment:
could we add a `Key()` method? so callers don't need to know about this mapping btw platform returned fields and agent slang?